internal/edittree/edittree.go

```diff
@@ -0,0 +1,1135 @@
+package edittree
```

Comment:
[nit] copyright header? I know we're not consistent about it, I suppose, but it can't hurt to include it.

Reply:
I wonder if there is a way for us to add the copyright banner check to CI somehow? At any rate, I've rolled the copyright headers into my latest changes. Thanks for catching that! :slightly_smiling_face: